snuba/datasets/factory.py

```diff
@@ -9,13 +9,14 @@
 DATASETS_IMPL: MutableMapping[str, Dataset] = {}
 DATASETS_NAME_LOOKUP: MutableMapping[Dataset, str] = {}
 
-DEV_DATASET_NAMES: Set[str] = {"metrics"}
+DEV_DATASET_NAMES: Set[str] = set()
 
 DATASET_NAMES: Set[str] = {
     "discover",
     "events",
     "groupassignee",
     "groupedmessage",
+    "metrics",
```

Comment:
This will not work until we have cluster declared in the prod and single tenant config.
Snuba will refuse to run healthchecks because it will not find a cluster for the storages associated to the dataset.
So we need the cluster in prod and accessible first.